octree/include/pcl/octree/octree_pointcloud_adjacency.h

```diff
@@ -247,9 +247,8 @@ namespace pcl
 
 }
 
-//#ifdef PCL_NO_PRECOMPILE
+// Note: Don't precompile this octree type to speed up compilation. It's probably rarely used.
```

Comment:
It's being instantiated [here](https://github.com/PointCloudLibrary/pcl/blob/master/segmentation/src/supervoxel_clustering.cpp#L172-L174). Not really where I would look for it normally. I would say for now leave it as it was, unless there's a solid reason to not precompile it.